### PR TITLE
Documentation for new desired cap: calendarAccessAuthorized

### PR DESCRIPTION
**Pull request by dpgraham**

### CODE REVIEW

**Comment by triager** [comment]
Can one of the admins verify this patch?

**Comment by triager** [comment]
Can one of the admins verify this patch?

File: docs/en/writing-running-appium/caps.md
Comment (jlipps) [inline docs/en/writing-running-appium/caps.md]: I think it would be a good idea to say what the default case is if someone sends in a value which is neither truthy nor false, or doesn't use the cap at all.

Comment (dpgraham) [inline docs/en/writing-running-appium/caps.md]: Done. I added a note that if it's non of the above the authorizationStatus of the calendar will remain unchanged.

**Review by jlipps — APPROVED** [review]
cool, this looks good to merge then once the dependencies are merged.

**Pull request merged**
